peps/pep-0819.rst

```diff
@@ -0,0 +1,347 @@
+PEP: 819
+Title: JSON Package Metadata
+Author: Emma Harper Smith <emma@python.org>
+PEP-Delegate: Paul Moore
+Discussions-To: Pending
+Status: Draft
+Type: Standards Track
+Topic: Packaging
+Created: 18-Dec-2025
+Post-History: Pending
+
+
+Abstract
+========
+
+Python package metadata ("core metadata") was first defined in :pep:`241` to
+use :rfc:`822` email headers to encode information about packages. This was
+reasonable at the time; email messages were the only widely used, standardized
+text format that had a parser in the standard library at the time. However,
+issues with handling different encodings, differing handling of line breaks,
+and other differences between implementations have caused numerous packaging
+bugs. To resolve these issues, this PEP proposes introducing
+`Javascript Object Notation (JSON) <https://www.json.org/json-en.html>`_
+encoded core metadata and wheel file format metadata files in Python packages.
+
+
+Motivation
+==========
+
+The email message format has a number of complexities and limitations which
+reduce its utility as a portable textual interchange format for packaging
+metadata. Due to the :mod:`email` parser requiring configuration changes to
+properly generate valid core metadata, many projects do not use the
+:mod:`!email` module and instead generate core metadata in a custom manner.
+There are many pitfalls with generating email headers that these custom
+generators can hit. First, core metadata fields may contain newlines in the
+value of fields. These newlines must be handled properly to "unfolded" multiple
+lines per :rfc:`822`. One particularly difficult to encode field is the
+``Description`` field, which may contain newlines and indentation. To encode
+the field in email headers, CRLF line breaks must be followed by seven (7)
+spaces and a pip ("|") character. While ``Description`` may now be encoded in
+the message body, similar escaping issues occur for the ``Author`` and
+``Maintainer`` fields. Improperly escaped newlines can lead to missing,
+partial, or invalid core metadata. Second, as discussed in the
+`core metadata specifications <https://packaging.python.org/specifications/core-metadata/>`__:
+
+.. epigraph::
```

Comment:
Per [docs](https://docutils.sourceforge.io/docs/ref/rst/directives.html#epigraph), this has no argument, so should be followed by a blank line
```suggestion
.. epigraph::

```